examples/matlab/getting_started.m

```diff
@@ -1,7 +1,7 @@
 % pyversion /usr/bin/python3
 % py.sys.path
 
-clear all
+clearvars
```

Comment:
why this change?

Reply:
Recent Matlab versions underline `clear all` and suggest to use `clearvars` instead.
It is supposed to be faster, I think.